app/controllers/projects_controller.rb

```diff
@@ -4,6 +4,8 @@ class ProjectsController < ApplicationController
   before_action :ensure_owner_or_admin, only: [ :edit, :update, :destroy, :volunteers ]
   before_action :set_filters_open, only: :index
   before_action :set_projects_query, only: :index
+  before_action :set_global_announcements, only: [ :index, :volunteered, :own, :show, :new, :create ]
+  before_action :show_global_announcements, only: [ :index, :volunteered, :own, :show, :new, :create ]
```

Comment:
Same with this one. We're looking for this:
1. Toggle `@show_global_announcements` to `true` for any request Performed in `ApplicationController`
2. Toggle it to false in `OffersControlller` (for any method in that controller IMHO)